src/main.cpp

```diff
@@ -6,14 +6,40 @@
 /*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/11/24 19:31:04 by mcourtoi          #+#    #+#             */
-/*   Updated: 2023/11/24 19:31:43 by mcourtoi         ###   ########.fr       */
+/*   Updated: 2024/02/28 17:35:19 by mcourtoi         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
-#include "irc.hpp"
+#include "ft_irc.hpp"
+#include <cstdlib>
+#include "class/Server.hpp"
+#include <cstdio>
+#include "IrcMessage.hpp"
 
-int main(void)
+/**
+ * @brief Launch irc server
+ * 
+ * // TODO : create a checker for the arguments
+ * // TODO : create a catcher for exceptions
+ * 
+ * Call the create server function who init the socket of the server and create the epoll loop
+ * Try to catch a possible exception by create_server()
+ * 
+ * @param ac 
+ * @param av 
+ * @return int 
+ */
+
+int main(int ac, char **av)
 {
-	std::cout << "hello world!\n";
+	if (ac != 3)
+		std::cout << "incorrect number of args\n";
```

Comment:
Why don't you leave the program then??
+
You should output this error message on `std::cerr` instead.

Reply:
Indeed